openupgrade_scripts/scripts/hr_holidays/15.0.1.5/post-migration.py

```diff
@@ -30,62 +30,21 @@ def _check_allocation_id(self):
 
 
 def _fill_hr_leave_type_holiday_allocation_id(env):
-    Allocation = env["hr.leave.allocation"].with_context(active_test=False)
-    employee_leaves = (
-        env["hr.leave"]
-        .with_context(active_test=False)
-        .search(
-            [
-                ("holiday_status_id.requires_allocation", "=", "yes"),
-                ("holiday_type", "=", "employee"),
-                ("date_from", "!=", False),
-                ("date_to", "!=", False),
-            ]
-        )
+    leaves = env["hr.leave"].search(
+        [
+            ("holiday_status_id.requires_allocation", "=", "yes"),
+            ("date_from", "!=", False),
+            ("date_to", "!=", False),
+        ]
```

Comment:
@vietphuong10 cần map từ xin nghỉ gần nhất
Giả xử có xin nghỉ ngày 1 tháng 1, ngày 2 tháng 2
Có cấp phát từ 1 tháng 1, từ 1 tháng 2

Nếu map từ quá khứ, thì cả 2 xin nghỉ sẽ map vào cấp phát từ 1 tháng 1
Mong đợi là xin nghỉ 2 tháng 2 map vào cấp pháp 1 tháng 2, nên cần map từ tương lại về quá khứ, do `date_to` cấp phát là null hết rồi
```suggestion
        ], order="date_to desc"
```

Reply:
model này sắp xếp `_order = "date_from desc"` rồi nên chỗ này không cần thêm nữa anh ạ

Reply:
ok em